<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Input</class>
 <widget class="QWidget" name="Input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 218, 218);
color: rgb(0, 0, 0);
font: 300 italic 17pt &quot;Ubuntu&quot;;</string>
  </property>
  <widget class="QPushButton" name="bnt_send">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>415</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">border: 2px solid black;</string>
        </property>
        <property name="text">
         <string>Введите возраст животного (int)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spb_age">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">border: 2px solid black;</string>
        </property>
        <property name="text">
         <string>Введите массу животного (double)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dspb_weight">
        <property name="maximum">
         <double>100000.990000000005239</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">border: 2px solid black;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите тип питания:&lt;br/&gt;'H' для травоядного (от слова &amp;quot;herbivore&amp;quot;),&lt;br/&gt;'C' для хищника (от слова &amp;quot;carnivore&amp;quot;),&lt;br/&gt;'O' для всеядного (от слова &amp;quot;omnivore&amp;quot;),&lt;br/&gt;или другие.  (char)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_dietType">
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">border: 2px solid black;</string>
        </property>
        <property name="text">
         <string>Это животное мужского пола (bool)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Мужского</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">border: 2px solid black;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите название разновидности (массив char, maxSize = 20)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_species">
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Часов сна за последнюю неделю (массив int, maxSize = 7)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QSpinBox" name="spb_1">
        <property name="maximum">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spb_2">
        <property name="maximum">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spb_3">
        <property name="maximum">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spb_4">
        <property name="maximum">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spb_5">
        <property name="maximum">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spb_6">
        <property name="maximum">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spb_7">
        <property name="maximum">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
